src/__tests__/setup.ts

```diff
@@ -20,6 +20,7 @@ jest.mock('../config/oauthConfig', () => ({
 }))
 
 // Mock the MySQL connection
+/*
```

Comment:
L22: Do you still need this commented out code?

Reply:
nope definitely not. Good catch!